forch/faucet_state_collector.py

```diff
@@ -1016,3 +1022,7 @@ def _get_egress_port(self, switch):
             if port_attr.get('type') == 'egress':
                 return port
         return None
+
+    def add_placement_callback(self, callback):
```

Comment:
As written, this should be set_placement_callback -- add_ would be a list of callbacks

Reply:
Done